EIPS/eip-7570.md

```diff
@@ -0,0 +1,65 @@
+---
+eip: 7570
```

Comment:
```suggestion
eip: 7605
```
Assigning next sequential EIP/ERC/RIP number.  7570 has already been assigned to an ERC.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
Noted, thanks.  Just pushed